e2e/connect-config-s3.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "eventbridge-e2e",
```

Comment:
question: do you also want to give the connector a different name? IMHO not needed right now because you don't parallelize, right? But might help to easily distinguish instances (assert) in your code?